<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Media_Player</class>
 <widget class="QWidget" name="Media_Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>428</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Media Player</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QFrame" name="button_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_back">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons/icons.qrc">
          <normaloff>:/buttons/previous-track.png</normaloff>:/buttons/previous-track.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_stop">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons/icons.qrc">
          <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_play_pause">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons/icons.qrc">
          <normaloff>:/buttons/pause-play.png</normaloff>:/buttons/pause-play.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_forward">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Icons/icons.qrc">
          <normaloff>:/buttons/next-track.png</normaloff>:/buttons/next-track.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="album_frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
	background-color: pink;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QSlider" name="volume_slider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lb_song_time">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lb_song_title">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_back</tabstop>
  <tabstop>pb_stop</tabstop>
  <tabstop>pb_play_pause</tabstop>
  <tabstop>pb_forward</tabstop>
 </tabstops>
 <resources>
  <include location="Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
